File: python/1.py
import json

with open('1-input.json') as input:
    data = json.load(input)

def findPair(data):
    for x in data["input"]:
        for y in data["input"]:
            if x + y == 2020:
                return [x, y]


ans1 = findPair(data)
# print(ans1, ans1[0] * ans1[1])


def findThree(data):
    for x in data["input"]:
        for y in data["input"]:
            for z in data["input"]:
                if x + y + z == 2020:
                    return [x,y,z]

ans2 = findThree(data)
print(ans2, ans2[0] * ans2[1] * ans2[2])

File: js/1.js
const input = require('./1-input.js')

let ans1;

for (let i = 0; i < input.length; i++) {
    for (let j = 0; j < input.length; j++) {
        if ((input[i] + input[j]) === 2020) {
            ans1 = [i, j]
        }
    }
}
console.log(input[ans1[0]], input[ans1[1]])
console.log(input[ans1[0]] * input[ans1[1]])

let ans2;

for (let i = 0; i < input.length; i++) {
    for (let j = 0; j < input.length; j++) {
        for (let k = 0; k < input.length; k++) {
            if ((input[i] + input[j] + input[k]) === 2020) {
                ans2 = [i, j, k]
            }
        }
    }
}
console.log(input[ans2[0]], input[ans2[1]], input[ans2[2]])
console.log(input[ans2[0]] * input[ans2[1]] * input[ans2[2]])